specification/machinelearning/resource-manager/Microsoft.MachineLearning/stable/2017-01-01/webservices.json

```diff
@@ -391,9 +408,6 @@
     "Resource": {
       "description": "Azure resource.",
       "x-ms-azure-resource": true,
-      "required": [
```

Comment:
SDK breaking change

Reply:
Per conversation I am adding required back to avoid breaking existing clients, despite the linter error :-)